File: solutions/45/sol.ts
const T: (n: number) => number = (n) => n * (n + 1) / 2

const P: (n: number) => number = (n) => n * (3 * n - 1) / 2

const H: (n: number) => number = (n) => n * (2 * n - 1)

for (let n = 1; n < 10; n++) {
  console.log(T(n), P(n), H(n));
}

File: solutions/28/sol.go
package main

import (
	"fmt"
)

func getAnswer(n int) []int {
	arr := []int{1}
	k := 3
	number := 1
	for k <= n {
		for i := 0; i < 4; i++ {
			number += k - 1
			arr = append(arr, number)
		}
		k += 2
	}
	return arr
}

func main() {
	fmt.Println(getAnswer(1))
	fmt.Println(getAnswer(3))
	fmt.Println(getAnswer(5))
	fmt.Println(getAnswer(7))
}
